From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Mon, 1 Sep 2025 04:05:23 +0200
Subject: build: Depend on GLib 2.86.0

As per previous change we now rely on a GLib version in which GioUnix
DesktopAppInfo implementation provides all the methods, so let's bump
the glib (actually gio) dependency for that.

As we need to be sure that the shell's typelib is rebuilt against newer
Gio, or updating GLib before than the shell would break us.

See: https://gitlab.gnome.org/GNOME/glib/-/issues/3776#note_2545887
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 56f1647..6aa77e6 100644
--- a/meson.build
+++ b/meson.build
@@ -21,7 +21,7 @@ libmutter_test_pc = 'libmutter-test-' + mutter_api_version
 ecal_req = '>= 3.33.1'
 eds_req = '>= 3.33.1'
 gcr_req = '>= 3.90.0'
-gio_req = '>= 2.85.1'
+gio_req = '>= 2.86.0'
 gjs_req = '>= 1.85.90'
 gtk_req = '>= 4.0'
 mutter_req = '>= 49.rc'
